Swiftfin tvOS/Views/UserSignInView/UserSignInView.swift

```diff
@@ -109,104 +107,121 @@ struct UserSignInView: View {
 
     @ViewBuilder
     private var publisUsersSection: some View {
-        Section(L10n.publicUsers) {
+        Section {
             if viewModel.publicUsers.isEmpty {
                 L10n.noPublicUsers.text
                     .font(.callout)
                     .foregroundColor(.secondary)
                     .frame(maxWidth: .infinity)
             } else {
-                ForEach(viewModel.publicUsers, id: \.id) { user in
-                    PublicUserRow(
-                        user: user,
-                        client: viewModel.server.client
-                    ) {
-                        username = user.name ?? ""
-                        password = ""
-                        focusedTextField = 1
+                HStack {
+                    Grid {
+                        GridRow {
+                            ForEach(viewModel.publicUsers, id: \.id) { user in
+                                PublicUserGrid(
+                                    user: user,
+                                    client: viewModel.server.client
+                                ) {
+                                    username = user.name ?? ""
+                                    password = ""
+                                    focusedTextField = 1
+                                }
+                            }
+                        }
                     }
                 }
             }
+        } header: {
+            Text(L10n.publicUsers)
+                .font(.title3)
+                .frame(maxWidth: .infinity, alignment: .center)
         }
     }
 
     var body: some View {
-        VStack {
-            HStack {
-                Spacer()
-
-                if viewModel.state == .signingIn {
-                    ProgressView()
+        ZStack(alignment: .center) {
+            VStack {
+                HStack {
+                    Image(.jellyfinBlobBlue)
+                        .resizable()
+                        .aspectRatio(contentMode: .fit)
+                        .frame(height: 100)
+                        .edgePadding()
                 }
-            }
-            .frame(height: 100)
-            .overlay {
-                Image(.jellyfinBlobBlue)
-                    .resizable()
-                    .aspectRatio(contentMode: .fit)
-                    .frame(height: 100)
-                    .edgePadding()
-            }
 
-            HStack(alignment: .top) {
-                VStack(alignment: .leading) {
-                    signInSection
+                VStack {
```

Comment:
Thank you for fixing the buttons, but this needs to be in a `HStack` like before. This is a deliberate design decision for these being different sections, and they should be equally sized. Additionally, the public user section can be of any size and requiring to scroll through them isn't desirable.